docs/src/modules/components/AppWrapper.js

```diff
@@ -1,207 +0,0 @@
-
-  React.useEffect(() => {
-    document.cookie = `codeVariant=${codeVariant};path=/;max-age=31536000`;
-  }, [codeVariant]);
-
-  return codeVariant;
-}
-
-function Tracking() {
-  const dispatch = useDispatch();
-  const { options } = useSelector(state => ({
-    options: state.options,
-  }));
-
-  const codeVariant = usePersistCodeVariant(options.codeVariant, nextCodeVariant =>
-    dispatch({ type: ACTION_TYPES.OPTIONS_CHANGE, payload: { codeVariant: nextCodeVariant } }),
-  );
-
-  React.useEffect(() => {
-    window.ga('set', 'dimension1', codeVariant);
-  }, [codeVariant]);
-
-  React.useEffect(() => {
-    window.ga('set', 'dimension2', options.userLanguage);
-  }, [options.userLanguage]);
-
-  React.useEffect(() => {
-    // Remove the server-side injected CSS.
-    const jssStyles = document.querySelector('#jss-server-side');
-    if (jssStyles) {
-      jssStyles.parentNode.removeChild(jssStyles);
-    }
-  }, []);
-
-  return null;
-}
-
-// Inspired by
-// https://developers.google.com/web/tools/workbox/guides/advanced-recipes#offer_a_page_reload_for_users
-function forcePageReload(registration) {
-  // console.log('already controlled?', Boolean(navigator.serviceWorker.controller));
-
-  if (!navigator.serviceWorker.controller) {
-    // The window client isn't currently controlled so it's a new service
-    // worker that will activate immediately.
-    return;
-  }
-
-  // console.log('registration waiting?', Boolean(registration.waiting));
-  if (registration.waiting) {
-    // SW is waiting to activate. Can occur if multiple clients open and
-    // one of the clients is refreshed.
-    registration.waiting.postMessage('skipWaiting');
-    return;
-  }
-
-  function listenInstalledStateChange() {
-    registration.installing.addEventListener('statechange', event => {
-      // console.log('statechange', event.target.state);
-      if (event.target.state === 'installed' && registration.waiting) {
-        // A new service worker is available, inform the user
-        registration.waiting.postMessage('skipWaiting');
-      } else if (event.target.state === 'activated') {
-        // Force the control of the page by the activated service worker.
-        window.location.reload();
-      }
-    });
-  }
-
-  if (registration.installing) {
-    listenInstalledStateChange();
-    return;
-  }
-
-  // We are currently controlled so a new SW may be found...
-  // Add a listener in case a new SW is found,
-  registration.addEventListener('updatefound', listenInstalledStateChange);
-}
-
-async function registerServiceWorker() {
-  if (
-    'serviceWorker' in navigator &&
-    process.env.NODE_ENV === 'production' &&
-    window.location.host.indexOf('material-ui.com') <= 0
-  ) {
-    // register() automatically attempts to refresh the sw.js.
-    const registration = await navigator.serviceWorker.register('/sw.js');
-    // Force the page reload for users.
-    forcePageReload(registration);
-  }
-}
-
-function AppWrapper(props) {
```

Comment:
Inline the file directly into _app.js. As long as the file is less than 1k LOCs (~400 here), we should be fine.